src/config/configTypes.ts

```diff
@@ -10,6 +10,7 @@ interface RepomixConfigBase {
     removeEmptyLines?: boolean;
     topFilesLength?: number;
     showLineNumbers?: boolean;
+    copyToClipboard?: boolean;
```

Comment:
Could you also add documentation for the `copyToClipboard` option in the README? It should go under the `output.showLineNumbers` entry in the Configuration section:

```markdown
|`output.copyToClipboard`| Whether to copy the output to system clipboard in addition to saving the file |`false`|
```

Reply:
Thank you for pointing it out! I apologize for not placing it in the documentation initially.